### PR TITLE
VACMS-9242 Benefits Hubs meta title length

### LINKED ISSUE #9242: Allow Benefits Hub and Benefits Detail <title> tags to exceed 70 characters
## User Story or Problem Statement

As a Benefits hub editor, i want to set page meta-titles longer than 70 characters, on a case by case basis. 

### Background

The [content style guide](https://design.va.gov/content-style-guide/seo#title-tags) specifies 70 characters or less for the two Benefits Hub content types. 

Occasionally, editors want to exceed that, and have page meta-titles match the h1 on the page. See https://github.com/department-of-veterans-affairs/va.gov-team/issues/41185 w/r/t to the [View or change your dependents on your VA disability benefits](https://www.va.gov/view-change-dependents) page. 

<details><summary>Screenshot</summary>

Edit_Benefits_Detail_Page_View_or_change_dependents_on_your_VA_disability_benefits___VA_gov_CMS

</details>

## Acceptance Criteria
- [ ] Metatag title field is removed on Benefits Hub Detail Page and Benefits Hub Landing Pages and are populated from default meta tags settings with the pattern `[node:title] | Veterans Affairs`  

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [x] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description

Closes #9242

## Testing done
manual, automatic

## Screenshots
image

image

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As CMS editor
1. Add or edit a Benefits Hub detail page (`node/3014/edit`)
   - [ ] Validate that the meta title field has been removed
   - [ ] Validate that the meta title matches the page title on [node view](https://cms-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/life-insurance) by viewing the source
   - [ ] Validate the [default meta tags](https://pr9246-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/admin/config/search/metatag/node__page) are set to use the pattern `[node:title] | Veterans Affairs` 
2. Add or edit a Benefits Hub landing page (`/node/77/edit`)
   - [ ] Validate that the meta title has been removed
   - [ ] Validate that the meta title matches the page title on [node view](https://cms-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/coronavirus-veteran-frequently-asked-questions) by viewing the source
   - [ ] Validate the [default meta tags](https://pr9246-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/admin/config/search/metatag/node__landing_page) are set to use the pattern `[node:title] | Veterans Affairs`
3. [Release the content](https://pr9246-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/admin/content/deploy) if no release is available
   - [ ] Validate that the meta title is populated on the detail [frontend](https://web-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/life-insurance/)
   - [ ] Validate that the meta title is populated on the landing page [frontend](https://web-i5fjotmvzy4e7fy1ylvmke58ukbhyiuc.ci.cms.va.gov/coronavirus-veteran-frequently-asked-questions/)
4. Then validate Acceptance Criteria from issue
   - [ ] Metatag title field is removed on Benefits Hub Detail Page and Benefits Hub Landing Pages and are populated from default meta tags settings with the pattern `[node:title] | Veterans Affairs` 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by kevwalsh — APPROVED** [review]
Other than the flag fields, this LGTM. Sitewide content team signed off on this change and its immediate impact on up to 350 Benefits Hub nodes.

File: config/sync/core.entity_view_display.node.landing_page.support_services_listing.yml [hunk 1]
```diff
@@ -86,6 +85,16 @@ content:
     third_party_settings: {  }
     weight: 3
     region: content
+  flag_email_node:
```
[the remark this answers is not in the document]
Comment (msbtterswrth) [reply config/sync/core.entity_view_display.node.landing_page.support_services_listing.yml:88]: ah i think that must've came from main which might be more updated locally than here. Should I merge main in and see what happens, or take this out?

[the remark this answers is not in the document]
Comment (swirtSJW) [reply config/sync/core.entity_view_display.node.landing_page.support_services_listing.yml:88]: I'm looking into this.  It is coming from the messages stack but I want to make sure it is benign.

[the remark this answers is not in the document]
Comment (swirtSJW) [reply config/sync/core.entity_view_display.node.landing_page.support_services_listing.yml:88]: Yah these are benign.   The flags are disabled 
image

These config will start appearing on future saves of other content types.  They are one of those configs that do not get added on config export unless there is some other change causing the content type to export.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Overall looks good.
We are missing a hook_update_n to clear out the data from the field storage though (assuming this field is used nowhere else).  Take a look in va_gov_db.install for examples or hit @ethanteague to show you how.

**Review by ethanteague — APPROVED** [review]
Looks good :+1:

**Pull request merged**